QRealTime_dialog.py

```diff
@@ -195,14 +201,24 @@ def sendForm(self,xForm_id,xForm):
         if form_key:
```

Comment:
This condition was kept to check if form already exist with given name. 
Did you check supported methods like 'put' for updating the existing form?

Reply:
Sir, I have made changes to check if the form already exists with given name and updating it using a PATCH request in the new commit. The sync option also works now.